### ISSUE TITLE
Migrate from mLab MongoDB add-on to Postgres add-on

### ISSUE THREAD

**Issue by satterly** [alerta/heroku-api-alerta#3]
The [mLab MongoDB add-on is being discontinued](https://devcenter.heroku.com/changelog-items/1823) and there doesn't seem to be any free plans for MongoDB from other providers so migrate to [Postgres on Heroku](https://www.heroku.com/postgres).

**Comment by satterly** [comment]
Backup and destroy the mLab add-on ...

```
$ mongodump -h ds029007.mlab.com:29007 -d heroku_12345678 -u heroku_12345678 -p random_password -o dump-dir
$ heroku addons:destroy mongolab
```

**Comment by satterly** [comment]
Add Postgres add-on and init the database on first run. Fixed by alerta/alerta@52b94fb314e92db36ac04c4afd72fc9bb460b040
